api/src/test/resources/ORM_O01.hl7

```diff
@@ -1,5 +1,5 @@
 MSH|^!\&||LNSP|||20180330150703.345+{time_offset}00||ORM^O01^ORM_O01|2018033015070300000|D|2.5
```

Comment:
Those fields are required for OpenELIS messages, and it was easy to include them here as well. I believe it may stay this way, but we should ask @pgesek . What do you think?